src/utils/api.js

```diff
@@ -1,13 +1,32 @@
 import axios from "axios";
 
 const server = axios.create({
-    baseURL: "https://ajai-back-end-project.onrender.com/api",
-  });
+  baseURL: "https://ajai-back-end-project.onrender.com/api",
+});
 
-  export function listOfReviews(){
-return server.get(`/reviews`).then((response)=>{
-return response.data.review.rows
+export function listOfReviews() {
+  return server.get(`/reviews`).then((response) => {
+    return response.data.review.rows;
+  });
+}
 
-})
+export function getReview(id) {
+  return server.get(`/reviews/${id}`).then((response) => {
+    return response.data;
+  });
+}
```

Comment:
Good work with this extra api function call Ajai - this is the correct thing to rather, rather than getting all reviews and then filtering them, this keeps the data up to date, well done!